plugins/org.eclipse.papyrus.uml.diagram.sequence.runtime/src/org/eclipse/papyrus/uml/diagram/sequence/runtime/internal/edit/parts/LifelineHeaderEditPart.java

```diff
@@ -45,6 +46,15 @@ public LifelineHeaderEditPart(View view) {
 		super(view);
 	}
 
+	@Override
+	public Command getCommand(Request _request) {
+		if ("connection end".equals(_request.getType())) {
+			toString();
```

Comment:
This if seems to be a left-over from debugging? If yes, please remove it.

Reply:
Yes, sorry, I will remove it